apps/angular/src/app/features/anime/anime-table.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, TrackByFunction } from '@angular/core';
+import { Anime } from '@js-camp/core/models/anime';
+import { map, Observable } from 'rxjs';
+
+import { AnimeService } from '../../../core/services/anime.service';
+
+/** Table for displaying the anime list. */
+@Component({
+  selector: 'camp-anime-table',
+  templateUrl: './anime-table.component.html',
+  styleUrls: ['./anime-table.component.css'],
+})
+export class AnimeTableComponent {
+
+  /** Current anime list. */
+  public animeList$ = new Observable<readonly Anime[]>();
+
+  /** Table column names. */
+  public readonly tableColumns = ['image', 'title', 'aired start', 'status', 'type'];
+
+  public constructor(
+    private readonly animeService: AnimeService,
+  ) {
+    this.animeList$ = this.animeService.getAnime().pipe(
+      map(result => result.results),
+    );
+  }
```

Comment:
There is not need to initialize `animeList$` two times.
```suggestion
  /** Current anime list. */
  public readonly animeList$: Observable<readonly Anime[]>;

  /** Table column names. */
  public readonly tableColumns = ['image', 'title', 'aired start', 'status', 'type'];

  public constructor(
    private readonly animeService: AnimeService,
  ) {
    this.animeList$ = this.animeService.getAnime().pipe(
      map(result => result.results),
    );
  }
```